src/pages/FactoryLoader/index.tsx

```diff
@@ -0,0 +1,256 @@
+      if (this.state.activeTabKey === LoadFactoryTabs.Progress) {
+        return;
+      }
+      this.alert = { variant, title };
+      this.setState({ alertVisible: true });
+      if (showAlertTimer) {
+        clearTimeout(showAlertTimer);
+      }
+      showAlertTimer = setTimeout(() => {
+        this.setState({ alertVisible: false });
+      }, variant === AlertVariant.success ? 2000 : 10000);
+    };
+    this.hideAlert = (): void => this.setState({ alertVisible: false });
+    // Prepare showAlert as a callback
+    if (this.props.callbacks && !this.props.callbacks.showAlert) {
+      this.props.callbacks.showAlert = (variant: AlertVariant, title: string) => {
+        this.showAlert(variant, title);
+      };
+    }
+  }
+
+  public componentDidUpdate(): void {
+    const { currentStep, hasError } = this.props;
+
+    const current = this.wizardRef.current;
+    if (current && current.state && current.state.currentStep !== currentStep && !hasError) {
+      current.state.currentStep = currentStep;
+    }
+
+    if (!hasError && this.state.currentRequestError) {
+      this.setState({ currentRequestError: '' });
+    }
+  }
+
+  private getIcon(step: LoadFactorySteps, className = ''): React.ReactNode {
+    const { currentStep, hasError } = this.props;
+    if (currentStep > step) {
+      return (<React.Fragment>
+        <CheckCircleIcon className={className} color="green" />
+      </React.Fragment>);
+    } else if (currentStep === step) {
+      if (hasError) {
+        return <ExclamationCircleIcon className={className} color="red" />;
+      }
+      return (<React.Fragment>
+        <InProgressIcon className={`${styles.rotate} ${className}`} color="blue" />
+      </React.Fragment>);
+    }
+    return '';
+  }
+
+  private getSteps(): WizardStep[] {
+    const { currentStep, workspaceName, devfileLocationInfo, hasError } = this.props;
+
+    const getTitle = (step: LoadFactorySteps, title: string, iconClass?: string) => {
+      let className = '';
+      if (currentStep === step) {
+        className = hasError ? 'error' : 'progress';
+      }
+      return (
+        <React.Fragment>
+          {this.getIcon(step, iconClass)}
+          <span className={className}>{title}</span>
+        </React.Fragment>
+      );
+    };
+
+    return [
+      {
+        id: LoadFactorySteps.INITIALIZING,
+        name: getTitle(
+          LoadFactorySteps.INITIALIZING,
+          'Initializing',
+          'wizard-icon'),
+        canJumpTo: currentStep >= LoadFactorySteps.INITIALIZING,
+      },
+      {
+        name: getTitle(
+          LoadFactorySteps.LOOKING_FOR_DEVFILE,
+          'Looking for devfile',
+          'wizard-icon'),
+        steps: [
+          {
+            id: LoadFactorySteps.APPLYING_DEVFILE,
+            name: getTitle(
+              LoadFactorySteps.APPLYING_DEVFILE,
+              devfileLocationInfo ?
+                `Found ${devfileLocationInfo}, applying it` :
```

Comment:
Seems it does not work correctly in case of specifying devfile via direct content URL. To test I used http://localhost:3000/#/load-factory?url=https://raw.githubusercontent.com/che-incubator/che-dashboard-next/master/devfile.yaml
![Screenshot_20201109_154317](https://user-images.githubusercontent.com/5887312/98548804-a0a2c400-22a2-11eb-8c53-876a9850794a.png)

Reply:
I have done it
![Screenshot from 2020-11-10 18-00-27](https://user-images.githubusercontent.com/6310786/98801287-49276400-241a-11eb-9f47-fe1e332dcab2.png)

Reply:
![Screenshot from 2020-11-06 17-32-26](https://user-images.githubusercontent.com/6310786/98801447-78d66c00-241a-11eb-8bac-9f09407e0489.png)

Reply:
I see different 
![Screenshot_20201111_135018](https://user-images.githubusercontent.com/5887312/98808502-e4bdd200-2424-11eb-84b0-1f534d108e54.png)
which is better but warning should be an error.

Reply:
![Screenshot from 2020-11-13 12-16-45](https://user-images.githubusercontent.com/6310786/99061876-d1d50a00-25aa-11eb-8dda-c3b8a5594d32.png)